src/components/OverflowMenu.tsx

```diff
@@ -4,14 +4,14 @@ import { HeaderButtons, OverflowMenu as RNOverflowMenu } from 'react-navigation-
 import labels from '../constants/labels.json'
 
 interface OverflowMenuProps {
-  children: ReactNode
+  children: (ReactElement | ReactNode[]) & ReactNode
```

Comment:
What is this type about?

Reply:
It was the bug occuring on ios, what @f1sh1918 mentioned and fixed.